src/app/zap-templates/zcl/data-model/chip/replacable-monitoring-cluster.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <name>HEPA Filter Monitoring</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes and commands for monitoring HEPA filters in a device</description>
+    <code>0x0071</code>
+    <define>HEPA_FILTER_MONITORING</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="HEPA_FILTER_MONITORING" type="percent" min="0" max="100" writable="false" isNullable="false">Condition</attribute>
```

Comment:
This needs `optional="true"`, since it's feature-dependent, right?  Same for all the other clusters here.